packages/compartment-mapper/src/link.js

```diff
@@ -95,16 +95,11 @@ const makeExtensionParser = (
     ) {
       language = languageForModuleSpecifier[specifier];
     } else {
-      if (!has(languageForExtension, extension)) {
-        throw new Error(
-          `Cannot parse module ${specifier} at ${location}, no parser configured for extension ${extension}`,
-        );
-      }
-      language = languageForExtension[extension];
+      language = languageForExtension[extension] || `extension:${extension}`;
```

Comment:
I would also accept `|| extension` based on our out-of-band conversation yesterday.